export.html

```diff
@@ -0,0 +1,286 @@
+
+<p>Click <span class="bold">Download</span> in the Activity Window to download the archive.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/exportOMETIFFWeb.jpg" class="align-center" src="images/exportOMETIFFWeb.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Select the appropriate format to export the image as a JPEG, PNG or TIFF.</p>
+
+<p>In OMERO.insight, use File Finder in the Download window to select the destination.</p>
+
+<p>In OMERO.web the file will be downloaded according to your browser convention.</p>
+
+<h3>OMERO.insight</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/exportFormatsInsight.jpg" class="align-left" src="images/exportFormatsInsight.jpg" style="width: 75%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/exportFormatsWeb.jpg" class="align-center" src="images/exportFormatsWeb.jpg" style="width: 80%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">When an image or dataset is downloaded as a JPEG, PNG or TIFF, individual images for each channel and a combined-channel image are generated for the selected Z section or time point for the image(s). All generated images are contained in a ZIP archive.</p>
+</div><!-- End note -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</ol>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<h2><a class="anchor" id="batch"></a>Using the Batch Image Export script</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<ol class="step">
+
+<li><p class="first">An alternative way to export multiple selected images or an entire dataset is to use the <span class="bold">Batch Image Export</span> script.</p>
```

Comment:
The pdf has an extra line here - 'This script enables a wide range of parameters to be configured before batch exporting.'